### ISSUE TITLE
[FEATURE REQUEST] allow variable in filename in capture

### ISSUE THREAD

**Issue by pcause** [chhoumann/quickadd#336]
**Is your feature request related to a problem? Please describe.**
I want to use global hotkey and create a note with quick add and not have the destination be fixed

**Describe the solution you'd like**
Allow quickadd variables to be included in the file name spec foe the capture.  I can then use:

/vault/inbox/captures/{{VALUE:filename}}

and have quick add prompt me for the filename and then use the value i enter as the file name to be created/added to.

could be a DATE or VDATE or any of the variable types.

**Comment by jaymaleoca** [comment]
I could get a variable file name with the syntax `Folder/@{{value:file_A,file_B...}}.md`, and `@` here is a prefix I used for that group of file names-to show that it's also possible to include the static element. 

I have created file_A, file_B, ... already.

I think I'm referring to the same thing as you?

**Comment by ChitaGideon** [comment]
If we could use {{MACRO:MyMacro::Start}} in the filename , that could be more flexible

**Comment by pcause** [comment]
> I could get a variable file name with the syntax `Folder/@{{value:file_A,file_B...}}.md`, and `@` here is a prefix I used for that group of file names-to show that it's also possible to include the static element.
> 
> I have created file_A, file_B, ... already.
> 
> I think I'm referring to the same thing as you?

yes, that would be similar, but I want to be prompted, not select from a set of options, or use others of the variables

**Comment by bohorquezod** [comment]
Hey 👋🏻, i'm not native english speaker so it could be posible that i misunderstood the thread but im using this format `900 - 🗓️ Journal/{{VDATE:Enter the tasks date, YYYY-MM-DD}}.md` to dinamicaly create task and append it to a custom daily note and works sharp

<img width="661" alt="image" src="https://github.com/chhoumann/quickadd/assets/14042353/ce363e2c-df20-4e44-b5d2-a97cc3d4924d">


<img width="695" alt="image" src="https://github.com/chhoumann/quickadd/assets/14042353/550c253a-f84a-45b1-9962-260bbe2816e8">

**new file**
<img width="100" alt="image" src="https://github.com/chhoumann/quickadd/assets/14042353/9172a214-89c6-4b1f-ade7-9f24a0aaaa29">
